app/Notifications/SlackTagNotification.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Rogue\Notifications;
+
+use Rogue\Models\Post;
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Notification;
+use Illuminate\Notifications\Messages\SlackMessage;
+
+class SlackTagNotification extends Notification
+{
+    use Queueable;
+
+    public $post;
+
+    /**
+     * Create a new notification instance.
+     *
+     * @return void
+     */
+    public function __construct(Post $post)
+    {
+        $this->post = $post;
+    }
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @param  mixed  $notifiable
+     * @return array
+     */
+    public function via($notifiable)
+    {
+        return ['slack'];
+    }
+
+    /**
+     * Get the mail representation of the notification.
+     *
+     * @param  mixed  $notifiable
+     * @return \Illuminate\Notifications\Messages\MailMessage
+     */
+    public function toSlack($notifiable)
+    {
+        if (config('services.slack.url')) {
+            return (new SlackMessage)
+                ->from('Tongue Cat', ':tonguecat:')
```

Comment:
This is a lame suggestion, but would it make sense to have the post by "Rogue" instead of "Tongue Cat"? (Thinking that could help more people on staff associate what Rogue is used for.)